components/dashboard/src/admin/License.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { PageWithSubMenu } from "../components/PageWithSubMenu";
+import { adminMenu } from "./admin-menu";
+
+import { LicenseContext } from "../license-context";
+import { ReactElement, useContext, useEffect } from "react";
+import { getGitpodService } from "../service/service";
+
+import { ReactComponent as Alert } from "../images/exclamation.svg";
+import { ReactComponent as Success } from "../images/tick.svg";
+import { ReactComponent as Tick } from "../images/check.svg";
+import { ReactComponent as Cross } from "../images/x.svg";
+
+export default function License() {
+    const { license, setLicense } = useContext(LicenseContext);
+
+    useEffect(() => {
+        if (isGitpodIo()) {
+            return; // temporarily disable to avoid hight CPU on the DB
+        }
+        (async () => {
+            const data = await getGitpodService().server.adminGetLicense();
+            setLicense(data);
+        })();
+    }, []);
+
+    const featureList = license?.enabledFeatures;
+    const features = license?.features;
+
+    // if user seats is 0, it means that there is user limit in the license
+    const userLimit = license?.seats == 0 ? "Unlimited" : license?.seats;
+
+    const [licenseLevel, paid, msg, tick] = getSubscriptionLevel(
+        license?.plan || "",
+        license?.userCount || 0,
+        license?.seats || 0,
+        license?.fallbackAllowed || false,
+    );
+
+    return (
+        <div>
+            <PageWithSubMenu
+                subMenu={adminMenu}
+                title="License"
+                subtitle="License associated with your Gitpod Installation"
+            >
+                <div className="flex flex-row space-x-4">
+                    <Card className="bg-gray-800 dark:bg-white text-white dark:text-gray-400">
+                        <p className="text-white dark:text-black font-bold pt-4 text-sm"> {licenseLevel}</p>
+                        <p className="dark:text-gray-500">{paid}</p>
```

Comment:
**question**: Probably not the best line to add this comment, but curious what _Paid_ means under the _Professional_ license header. Could we use a more useful subtitle like the following?

> Expires on Dec 22, 2023

Not sure if this information is available though. If not, let's go with the existing chage.

Reply:
This info might be available in certain cases, like a `Trial` Licence from replicated. But yeah might not be available in the case of a professional licence. The amount they paid for the licence(as suggested in your initial design specs) is an information we currently cannot gather from the licence.

Reply:
May be in the case of Trial I can try and bring the expiration info, since the account is technically not paid for.

Reply:
updated it to reflect expires value for trial license